addons/website/views/snippets/s_popup.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_popup" name="Popup">
+    <div class="s_popup o_snippet_invisible" string="Popup">
+        <div class='modal s_popup_main popup_backdrop s_popup_center o_one_page bg-black-25 ' data-show_after="10" data-display="afterDelay">
+            <div class='modal-dialog modal-dialog-centered s_popup_frame modal-md'>
+                <div class='oe_structure modal-content s_popup_content oe_img_bg oe_custom_bg' style="background-color: #FFFFFF">
+                    <div class='s_popup_close o_we_no_overlay o_not_editable'>&amp;times;</div>
+                    <section class="s_banner parallax s_parallax_is_fixed pt96 pb96 s_parallax_no_overflow_hidden" data-scroll-background-ratio="1" style="background-image: none;">
```

Comment:
In fact, is there a visible effect of parallax in a modal? Otherwise maybe just remove the parallax stuff which will simplify the DOM